.github/container/Dockerfile.jax

```diff
@@ -45,15 +53,24 @@ ARG BASE_IMAGE
 FROM ${BASE_IMAGE} as runtime-image
 ARG SRC_PATH_JAX
 ARG SRC_PATH_XLA
+ARG SRC_PATH_TE
 ARG BUILD_DATE
 ENV BUILD_DATE=${BUILD_DATE}
 
 COPY --from=jax-builder ${SRC_PATH_JAX}-no-git ${SRC_PATH_JAX}
 COPY --from=jax-builder ${SRC_PATH_XLA}-no-git ${SRC_PATH_XLA}
+COPY --from=jax-builder ${SRC_PATH_TE}-no-git ${SRC_PATH_TE}
 
-RUN pip --disable-pip-version-check install ${SRC_PATH_JAX}/dist/*.whl && \
-    pip --disable-pip-version-check install -e ${SRC_PATH_JAX} && \
-    rm -rf ~/.cache/pip/
+RUN <<EOF
+# Transformer Engine installation dependencies
+pip install --no-cache-dir pybind11 ninja packaging
+# Install JAX + Transformer Engine
+NVTE_FRAMEWORK=jax pip --disable-pip-version-check --no-cache-dir install -e \
```

Comment:
I'm wondering what you think about setting `ENV NVTE_FRAMEWORK=jax`. Since we must set that environment variable to correctly install TE, it seems more friendly for re-installs. For example, when installing t5x next, it'll look something like `pip install -e /opt/transformer-engine -e /opt/t5x` and if `NVTE_FRAMEWORK` is set in the base container, we can save a few characters when installing

Reply:
Good point.

Reply:
The env variable isn't needed anymore if jax is already installed.
By default, the setup.py file try to import all the fw and will build for those installed.